lib/open_food_network/feature_toggle.rb

```diff
@@ -1,3 +1,7 @@
+require 'open_food_network/feature'
+require 'open_food_network/null_feature'
+require 'open_food_network/ga_feature'
```

Comment:
I want to move all the feature toggles code to lib/feature_toggle/ but doing it now will likely mess up with the work I'm doing in https://github.com/openfoodfoundation/openfoodnetwork/pull/6643. This is good enough to keep on going IMO, and I'll finish later on.